### ISSUE TITLE
Error 500

### ISSUE THREAD

**Issue by grigorovich-alex** [ITPrism/VirtualCurrency#7]
500 itprismUi not found on all virtual currency component pages in joomla adminpanel.
Installed latest version of ITprism Library 1.6, cache cleaned... Something wrong.

**Comment by forexic** [comment]
500 itprismUi not found . I keep receiving this error all the times . I have installed the latest version of Prism Library but still the component dose not work . Please let us know what to do.

**Comment by jhvanamson** [comment]
Same issue here with VC 1.2, ITPrism Lib 1.8, joomla 3.4.5, php 5.4.26, mysql 5.5.42-cll.
The only pages of the Virtual Currency component that work are the dashboard and plugins.
On the dashboard of VC, it thinks the library hasn't been installed yet, so in my case the problem it can't detect it.

**Comment by jhvanamson** [comment]
It looks like VC hasn't been developed anymore while the library has, so they are not compatible anymore. If this is the case, could supply us with a compatible version of the library? Or are you planning on an update for VC?

**Comment by ITPrism** [comment]
Hi,
I had not enough time to develop Virtual Currency last months.
However, I am planning to do it now because it is going to be one of my main tools in my next project.
So, I am going to publish updates frequently. 

I guess, I will be able to publish new version of VC 2.0 at the end of next week.

**Comment by jhvanamson** [comment]
Hi, would it be possible to get a version of the library that works with version 1.2? I just can't wait to start working with VC

**Comment by ITPrism** [comment]
Hi,
You can use old version of Prism library - ITPrism Library.

http://cdn.itprism.com/extensions/library/old/lib_itprism_v1.14.zip
http://cdn.itprism.com/extensions/library/old/lib_itprism_v1.15.zip

P.S.
I am planning to complete it to the end of this month.
You can start with v1 and when I am done, you will be able to upgrade to v2.

**Comment by jhvanamson** [comment]
Thank you!

**Comment by goranapo** [comment]
Is Prism library v2.0 published yet?

**Comment by ITPrism** [comment]
Yeah. I am working now to upgrade Virtual Currency to use Prism Library.

**Comment by ITPrism** [comment]
I have just upload the new version of [Virtual Currency v2.0] (http://cdn.itprism.com/extensions/virtualcurrency/pkg_virtualcurrency_v2.0-J3.5.zip). It will be published official on 29 February. If you would like, you can download and test it.

You have to install [Prism Library 1.9] (http://cdn.itprism.com/extensions/library/lib_prism_v1.9.zip) too.

Regards,
T

**Comment by Adgenodux** [comment]
Hi and thx for new versions...

If the 500 error is finaly gone, unfortunately, there are new errors.

The **Transactions** and **currency links** causes the following error:
`1054 Unknown column 'a.image' in 'field list' SQL=SELECT a.id, a.title, a.code, a.symbol, a.params, a.published, a.image, a.image_icon FROM `gcdb_vc_currencies` AS `a` `

The **Accounts** and **Plugins links** are working right.

The **Virtual Goods link** causes the following error:
`1054 Unknown column 'a.code' in 'field list' SQL=SELECT a.id, a.title, a.code, a.symbol, a.position FROM `gcdb_vc_realcurrencies` AS `a` WHERE a.id = 0`

The **Real Currencies link** open the manager page but display the following error just above the columns:
`Warning: Invalid argument supplied for foreach() in /var/www/administrator/components/com_virtualcurrency/views/realcurrencies/tmpl/default_body.php on line 13`

Hope it helps...

Regards...

**Comment by ITPrism** [comment]
I guess, you have installed old version and old tables still exist. 
You have to install new release over old one. That will create newest fields.

Alternatively, remove old tables manually before installation of new release.

**Comment by Adgenodux** [comment]
well, you're right...

following your advise, i've manualy erased old tables and after new install, looks better...

thanks for help... ;-)
